wait/walk_test.go

```diff
@@ -0,0 +1,87 @@
+package wait_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func TestWalk(t *testing.T) {
+	req := testcontainers.ContainerRequest{
```

Comment:
suggestion: I think we do not need the req, just the WaitingFor field

Reply:
You're correct, but I wanted the test to act as an example of its use as well, hence using request.